app/models/state_file_id_intake.rb

```diff
@@ -76,6 +77,8 @@
 #  index_state_file_id_intakes_on_spouse_state_id_id   (spouse_state_id_id)
 #
 class StateFileIdIntake < StateFileBaseIntake
+  self.ignored_columns = [:df_data_import_failed]
```

Comment:
Should we just add this line to the `StateFileBaseIntake` instead if its ignored on all the states intakes?

Reply:
oh great suggestion, will do!